tests/unit_tests/test_images.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+
+from pynwb.base import Images, ImageReferences
+from pynwb.image import GrayscaleImage
+
+from nwbinspector.checks.images import check_order_of_images_unique, check_order_of_images_len
+
+
+def test_check_order_of_images_unique():
+
+    imgs = [GrayscaleImage(name=f"image{i}", data=np.random.randn(10, 10)) for i in range(5)]
+    img_refs = ImageReferences(name="order_of_images", data=imgs + [imgs[0]])
+    images = Images(name="my_images", images=imgs, order_of_images=img_refs)
+
+    assert check_order_of_images_unique(images).message == "order_of_images should have unique values."
```

Comment:
Highly recommend testing against a full `InspectorMessage` output to ensure secondary information is parsed correctly to avoid bugs like that `None` issue you found last week. Also important for testing the `Importance` level matches what was intended.

Reply:
Thanks!